docs/language/search-syntax/README.md

```diff
@@ -125,97 +114,102 @@ describes ways to perform searches against only fields of a specific
 Sometimes you may need to search for sequences of multiple words or words that
 contain special characters. To achieve this, wrap your search term in quotes.
 
-Let's say we want to isolate the records containing the text
-`O=Internet Widgits` that we saw in the response to the previous example
-search. If typed bare as our Zed query, we'd experience two problems:
+Let's say we've noticed that a couple of the school names in our sample data
+include the string `Defunct=`. An attempt to enter this as a [bare word](#bare-word)
+search causes an error because the language parser interpreted this as the
+start of an attempted [field/value match](#fieldvalue-match) for a field named
+`Defunct`.
 
-1. The leading `O=` would be interpreted as the start of an attempted
-   [field/value match](#fieldvalue-match) for a field named `O`.
+#### Example:
+```mdtest-command zed-sample-data/edu/zson
+zq -z 'Defunct=' *.zson || true
```

Comment:
The `|| true` is only here to work around a current mdtest limitation. I either need to do something like this in the example command line to make it look like "success" or simply not protect it with mdtest. There's already another example in the docs that's unprotected for the same reason, so that motivated me to discuss it with @nwt and then I opened #2857. If that can't be addressed before we want to merge these docs, I could go ahead and leave the example unprotected and leave a note in the issue to circle back when we address #2857.